services/order.py

```diff
@@ -0,0 +1,45 @@
+from django.db import transaction
+from django.db.models import QuerySet
+from db.models import Ticket, Order, get_user_model, MovieSession
+
+
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str = None) -> None:
+
+    with transaction.atomic():
+
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+        tickets_instances = []
+        if date:
+            order.created_at = date
```

Comment:
Use transaction.atomic as decorator instead of context manager. It will make your code more readable